### PR TITLE
[Experiment] New WP Dashboard Widgets

### PR DESCRIPTION
**Pull request by KingYes**
Continue works from #13013

### CODE REVIEW

**Comment by rodolphebertozzo** [comment]
A lot of flaws in PHP lint

File: modules/dashboard-widgets/module.php [hunk 1]
```diff
@@ -0,0 +1,483 @@
...
+	public function __construct() {
+		$this->set_vars();
+
+		remove_action( 'welcome_panel', 'wp_welcome_panel' );
+
+		if ( 3 >= $this->recently_edited_query->post_count ) {
+			add_action( 'welcome_panel', [ $this, 'welcome_dashboard_widget_render' ], PHP_INT_MAX );
+		}
+
+		//add_action( 'admin_enqueue_scripts', [ $this, 'load_assets' ] );
```
Comment (rodolphebertozzo) [inline modules/dashboard-widgets/module.php:41]: Remove

File: modules/dashboard-widgets/module.php [hunk 2]
```diff
@@ -0,0 +1,483 @@
...
+	 * @var \WP_Query $recently_edited_query
+	 */
+	private $recently_edited_query;
+
+	public function get_name() {
+		return 'widgets';
+	}
+
+	public function __construct() {
+		$this->set_vars();
```
Comment (rodolphebertozzo) [inline modules/dashboard-widgets/module.php:33]: Why ?

Comment (KingYes) [reply modules/dashboard-widgets/module.php:33]: I just get this code and improve this. You can see it still in draft.

**Pull request closed without merging**